doc/users/next_whats_new/selector_improvement.rst

```diff
@@ -0,0 +1,39 @@
+Selectors improvement: rotation, aspect ratio correction and add/remove state
+-----------------------------------------------------------------------------
+
+The `~matplotlib.widgets.RectangleSelector` and
+`~matplotlib.widgets.EllipseSelector` can now be rotated interactively.
+The rotation is enabled or disable by striking the *r* key
+(default value of 'rotate' in *state_modifier_keys*) or by calling
+*selector.add_state('rotate')*.
+
+The aspect ratio of the axes can now be taken into account when using the
+"square" state. This is enable by specifying *use_data_coordinates='True'* when
```

Comment:
```suggestion
"square" state. This is enabled by specifying ``use_data_coordinates='True'`` when
```
As per my general comments, I think this could also do with some more explanation what is meant by "the axes ratio of the axes can be taken into account", i.e. something like "which uses the extents in data coordinates for defining the square aspect ratio".